content/docs/guides/10_angular_relay.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Send and Receive Messages Using Waku Relay With Angular v13
+date: 2022-02-15T09:00:00+01:00
+weight: 7
+---
+
+# Send and Receive Messages Using Waku Relay With Angular v13
+
+It is easy to use Waku Connect with Angular v13.
+In this guide, we will demonstrate how your Angular dApp can use Waku Relay to send and receive messages.
+
+Before starting, you need to choose a _Content Topic_ for your dApp.
+Check out the [how to choose a content topic guide](/docs/guides/01_choose_content_topic/) to learn more about content topics.
+For this guide, we are using a single content topic: `/relay-angular-chat/1/chat/proto`.
+
+# Setup
+
+Create a new Angular app:
+
+```shell
+npm install -g @angular/cli
+ng new relay-angular-chat
+cd relay-angular-chat
+```
+
+## `BigInt`
+
+Some of js-waku's dependencies use [`BigInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt)
+that is [only supported by modern browsers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#browser_compatibility).
+
+To ensure that Angular properly transpiles your webapp code, add the following configuration to the `package.json` file:
+
+```json
+{
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not ie <= 99",
+      "not android <= 4.4.4",
+      "not dead",
+      "not op_mini all"
+    ]
+  }
+}
+```
+
+## Polyfills
+
+A number of Web3 dependencies need polyfills.
+These must be explicitly declared when using webpack 5.
+
+The latest `Angular` version (v13) uses webpack 5.
+
+We will describe below a method to configure polyfills when using `Angular v13 / webpack v5`.
+This may not be necessary if you use webpack 4.
+
+Start by installing the polyfill libraries:
+
+```shell
+yarn add assert buffer crypto-browserify process stream-browserify
+```
+
+Then add the following code to `src/polyfills.ts`:
+
+```
+import * as process from 'process';
+(window as any).process = process;
+(window as any).global = window;
+global.Buffer = global.Buffer || require('buffer').Buffer;
+
+```
+
+Now tell Angular where to find these libraries by adding the following to `tsconfig.json`
+under `"compilerOptions"`:
+
+```json
+"paths": {
+  "assert": ["node_modules/assert"],
+  "buffer": ["node_modules/buffer"],
+  "crypto": ["node_modules/crypto-browserify"],
+  "stream": ["node_modules/stream-browserify"]
+},
```

Comment:
```suggestion
{
	  "paths": {
	    "assert": ["node_modules/assert"],
	    "buffer": ["node_modules/buffer"],
	    "crypto": ["node_modules/crypto-browserify"],
	    "stream": ["node_modules/stream-browserify"]
	  },
}
```

This is to avoid the red syntax highlight from hugo.